data/formats-data.js

```diff
@@ -1162,7 +1162,7 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	lickilicky: {
-		randomBattleMoves: ["wish", "protect", "bodyslam", "knockoff", "dragontail", "healbell", "swordsdance", "explosion", "earthquake", "powerwhip"],
+		randomBattleMoves: ["wish", "protect", "bodyslam", "knockoff", "dragontail", "healbell", "swordsdance", "explosion", "earthquake", "powerwhip", "thrash"],
```

Comment:
Thrash is a shit move.

Reply:
I thought about it but it could be good with own tempo considering own tempo + fatigue move mons are fairly decent in standard play